test/views/IndexViewSpec.scala

```diff
@@ -142,13 +141,6 @@ class IndexViewSpec extends SpecBase with ViewBehaviours {
           doc.select(Selectors.breadcrumbWithLink(1)).attr("href") shouldBe appConfig.vatOverviewUrl
         }
 
-        "have a link to external guidance which opens in a new tab" in {
```

Comment:
Do we not want to keep this in?

Reply:
The test can't find the selector as the content has been moved out of the index view so I assumed it wasn't needed in here and moved it to the helper.

Reply:
That makes sense - just thought that if someone were to remove the helper from the view in the controller then the content would be missing. Happy to leave it out.